java/test/apps/tests/AllTest.java

```diff
@@ -41,8 +41,10 @@ public static Test suite() {
         // all tests from here down in heirarchy
         TestSuite suite = new TestSuite("AllTest");  // no tests in this class itself
         // all tests from other classes
-        suite.addTest(jmri.PackageTest.suite());
-        suite.addTest(new junit.framework.JUnit4TestAdapter(apps.PackageTest.class));
+        suite.addTest(jmri.jmrit.logix.OBlockTest.suite());
+//        suite.addTest(jmri.jmrit.logix.PackageTest.suite());
+//        suite.addTest(jmri.PackageTest.suite());
+//        suite.addTest(new junit.framework.JUnit4TestAdapter(apps.PackageTest.class));
```

Comment:
In addition to Randall's comment about why the tests in question are being diabled, you are adding the test for Oblocks in the wrong location.

The oblocks test should be referenced in the PackageTest for the jmri.jmrit.logix, which also should not be referenced above ( you have it commented out, but it should not appear here at all).